File: JsonInflatorIOS/JsonInflator/Classes/utility/InflatorColorLookup.swift
//
//  InflatorColorLookup.swift
//  Json inflator Pod
//
//  Library utility: protocol for color lookup
//  Integrates with InflatorConvUtil to look up custom color references
//

public protocol InflatorColorLookup {
    
    func getColor(refId: String) -> UIColor?
    
}

File: JsonInflatorAndroid/JsonInflatorLib/src/main/java/com/crescentflare/jsoninflator/utility/InflatorColorLookup.java
package com.crescentflare.jsoninflator.utility;

import org.jetbrains.annotations.NotNull;
import org.jetbrains.annotations.Nullable;

/**
 * Json inflator utility: interface for color lookup
 * Integrates with InflatorMapUtil to look up custom color references
 */
@SuppressWarnings({"unused", "WeakerAccess"})
public interface InflatorColorLookup
{
    @Nullable
    Integer getColor(@NotNull String refId);
}
